app/views/layouts/_footer.html.erb

```diff
@@ -0,0 +1,8 @@
+
+    <footer class="footer">
+      <div class="container">
+        <div class="footer-copyright text-center">© <% Time.now.year %> Copyright:
```

Comment:
add the screenshot how it is showing in footer.

Reply:
![Screenshot from 2022-02-22 12-20-26](https://user-images.githubusercontent.com/82970871/155082119-a48ad63c-adbe-4abf-9026-5f7b84e867fa.png)